ansible/playbooks/roles/upgrade/tasks/kubernetes.yml

```
@@ -12,14 +12,26 @@
   import_tasks: kubernetes/get-kubelet-version.yml # sets kubelet_version
   delegate_to: "{{ groups.kubernetes_master[0] }}"
 
+- name: Check for upgrade flag file
+  stat:
+    path: "{{ kubernetes.upgrade_state_file_path }}"
+  register: lock_file_status
```

```suggestion
- name: Check for upgrade state flag file
  stat:
    path: "{{ kubernetes.upgrade_state_file_path }}"
  register: upgrade_state_file_status
```
I would use upgrade state file for consistency and avoid using `lock` term since it is often used for preventing running multiple instances at the same time (e.g. by yum or apt).